lib/core/utils/match-ancestry.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Check if two ancestries are identical
+ */
+function matchAncestry(ancestryA, ancestryB) {
```

Comment:
From what I can tell, this should already be available through axe.utils because it's exported from core/utils/index.js, which core/core.js imports everything from and makes available as axe.utils. 

I can add tests for it though.